.github/workflows/publish-java.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish package to Maven Central Repository
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
When I test this regex on online regex test site, it doesn't include v1.0, v0.1.
To include vx.x, you may use `v[0-9]+(.[0-9]+)?.[0-9]+`.